starknet_py/contract.py

```diff
@@ -256,26 +256,32 @@ def _account(self) -> BaseAccount:
     async def call_raw(
         self,
         block_hash: Optional[str] = None,
+        block_number: Optional[Union[int, Tag]] = None,
     ) -> List[int]:
         """
         Calls a method without translating the result into python values.
 
         :param block_hash: Optional block hash.
+        :param block_number: Optional block number.
         :return: list of ints.
         """
-        return await self._client.call_contract(call=self, block_hash=block_hash)
+        return await self._client.call_contract(
+            call=self, block_hash=block_hash, block_number=block_number
+        )
 
     async def call(
         self,
         block_hash: Optional[str] = None,
+        block_number: Optional[Union[int, Tag]] = None,
```

Comment:
What if we made at least `block_number` a keyword only one? 

Ideally there'd be a way to detect if `block_hash` was passed as a positional argument and show a warning, but that might not be possible.

Reply:
I'd rather keep both of them simple, non keyword only / both keyword.